src/Metal/MTLRenderCommandEncoder.cs

```diff
@@ -0,0 +1,23 @@
+﻿//using System;
+//using System.ComponentModel;
+//using System.Runtime.InteropServices;
+
+//using XamCore.Foundation;
+//using XamCore.ObjCRuntime;
+
+//namespace XamCore.Metal {
+//	public static class MTLRenderCommandEncoder_Extensions {
+//		public unsafe static void SetViewports (this IMTLRenderCommandEncoder This, MTLViewPort [] viewports)
```

Comment:
Whoops, commented it out when I tried binding directly to the array in the bindings instead (and saw that it tries to bind it wrong if you do that), apparently forgot to uncomment it again.  Will fix.